index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  
+   
+    <title>Portfolio Ameer Hamza</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div class="header-nav-phone">
+            <a href="/" class="header-logo">
+                <h3>
+                    Logo
+                </h3>
+            </a>
+            <button type="button" class="menu-btn">
+                <img src="./img/uni.svg" alt="menu link">
+            </button>
+            <nav class="header-nav-mobile hidden">
+                <button type="button" class="btn-close-nav">
+                    <img src="./img/cancel-ic.svg" alt="cancel icon">
+                </button>
+                <ul>
+                    <li>
+                        <a href="#works-id" class="link-portfolio">
+                            Portfolio
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#about-myself-id" class="link-about">
+                            About
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#footer-container-id" class="link-contact">
+                            Contact
+                        </a>
+                    </li>
+                </ul>
+            </nav>
+
+        </div>
+    </header>
+    <section class="headline">
+        <nav>
+            <ul class="header-nav-desktop">
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+            </ul>
+        </nav>
+        <div class="container container-flex">
+            <h1 class="header-title">
+                Hey there. I'm Ameer Hamza
+            </h1>
+            <h1>
+                <span>I'm a software developer</span>
+            </h1>
```

Comment:
- [ ] Please keep one `h1` tag for your page. It is better for your page to have just one h1 tag, this makes your website semantic and easily readable by browsers. :+1: